data/brands/shop/clothes.json

```diff
@@ -3200,6 +3200,17 @@
         "shop": "clothes"
       }
     },
+    {
+      "displayName": "H&M Kids",
+      "locationSet": {"include": ["gb"]},
```

Comment:
This is definitely in the UK. In e.g. Luxembourg, there categories seem to be Youth and Enfant, so I'm guessing they wouldn't be branded as "Kids"

Reply:
Found two more countries based on hits on OSM.

Reply:
> This is definitely in the UK. In e.g. Luxembourg, there categories seem to be Youth and Enfant, so I'm guessing they wouldn't be branded as "Kids"

This project's data is collected for OSM's environment, we also follow the "on the ground" rule.

Reply:
Yeah, hence I just added the ones from countries already present in OSM:
https://www.openstreetmap.org/search?query=h%26m%20kids